Makefile

```diff
@@ -836,6 +836,12 @@ trwiki_models: \
 trwiki_tuning_reports: \
 	tuning_reports/trwiki.wp10.md
 
+####################### Ukrainian Wikipedia #################################
+datasets/ukwiki.labelings.20200501.json:
+	./utility extract_labelings \
+		/mnt/data/xmldatadumps/public/ukwiki/20200501/ukwiki-20200501-pages-meta-history?*.xml-*.bz2 > $@
+
+
```

Comment:
Looks like we'll need some lines to balance the labels, extract text, and extract features.

Reply:
Working on that right now